x-pack/plugins/security_solution/server/lib/detection_engine/rules/utils.ts

```diff
@@ -102,7 +102,10 @@ export const transformFromAlertThrottle = (
   legacyRuleActions: LegacyRuleActions | null | undefined
 ): string => {
   if (legacyRuleActions == null || (rule.actions != null && rule.actions.length > 0)) {
-    if (rule.muteAll || rule.actions.length === 0) {
+    if (
+      rule.muteAll ||
+      (rule.actions.length === 0 && !(rule.params as { responseActions: unknown })?.responseActions)
```

Comment:
@marshallmain would this be better? I just have issue with typing this, I don't want to have a typequard for checking rule type === 'query' because I am not sure if this throttle isnt for other rule types too.

Reply:
The responseActions dependency on `throttle` is removed entirely now, right?

Reply:
yes, it should be removed already, unless I ommited anything by accident.